From 064ef546f22c90a0c006ea3d9faecf3e53dea5c6 Mon Sep 17 00:00:00 2001
From: Anton Arapov <anton@redhat.com>
Date: Tue, 6 Mar 2012 10:49:47 +0100
Subject: [PATCH 04/53] plugin: kerneloops: change the kerneloops url to
 report

  www.kerneloops.org is in transit. we will roll back this change once
the domain is ours.

Signed-off-by: Anton Arapov <anton@redhat.com>
Signed-off-by: Nikola Pajkovsky <npajkovs@redhat.com>
---
 src/plugins/report_Kerneloops.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/report_Kerneloops.xml.in b/src/plugins/report_Kerneloops.xml.in
index 75a9597..bc9c9ed 100644
--- a/src/plugins/report_Kerneloops.xml.in
+++ b/src/plugins/report_Kerneloops.xml.in
@@ -15,7 +15,7 @@
             <_label>Kerneloops URL</_label>
             <allow-empty>no</allow-empty>
             <_description>Oops server url</_description>
-            <default-value>http://submit.kerneloops.org/submitoops.php</default-value>
+            <default-value>http://kerneloops.descope.org/submitoops.php</default-value>
         </option>
     </options>
 </event>
-- 
1.7.11.2

